plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -172,6 +187,12 @@ func (s *AliyunCMS) Init() error {
 			len(s.Regions), strings.Join(s.Regions, ","))
 	}
 
+	// Check metric services
+	if len(s.MetricServices) == 0 {
+		s.MetricServices = []string{"cms"}
+		s.Log.Info("'metric_services' is not set. Metrics will be queried from the cms service")
```

Comment:
Please avoid this kind of logging! The default value should be documented in `sample.conf` and doesn't require a log line nobody reads...